package cms_8120;
option java_package = "com.hikvision.finance.fms.cms.hpp.csclient";
option java_outer_classname = "SystemConfigProto";
option java_generic_services = true;


message SysconfigInfo
{
    optional int32 nKey                     = 1;    // 参数key值
    optional string strValue                = 2;    // 参数值
    optional string strComment              = 3;    // 内容
}

// 请求信息
message ReqSysconfig
{
    optional int32 cmd                      = 1;    // CMD_DT_SYSTEM_CONFIG
}

// 客户端响应信息
message RspSysconfig
{
    optional int32 cmd                      = 1;
    optional int32 result                   = 2;    // 验证结果,0-验证通过，其它值验证失败
    optional string result_info             = 3;    // 信息
    repeated SysconfigInfo sysconfigInfo    = 4;
}

service SystemConfigService {
	rpc QuerySysconfigInfo(ReqSysconfig) returns (RspSysconfig);			// CMD_DT_SYSTEM_CONFIG
}